poleno/utils/misc.py

```diff
@@ -164,6 +164,25 @@ def guess_extension(content_type, default=u'.bin'):
         res = default
     return res
 
+def sanitize_filename(filename, content_type, default_base=u'attachment'):
+    u"""
+    Sanitize file name and adjust its extension according to given ``content_type``. Remove all
+    control characters (with ASCII code below 32) from the file name and shorten it to a maximum of
+    200 characters. If the resulting file name excluding the extension is empty, use
+    ``default_base`` instead of it. Adjust file extension if the given ``content_type`` differs from
+    the content type represented by the original file extension to a correct one.
```

Comment:
@viliambalaz Trochu som preformuloval docsctring k funkcii. Prva veta by mala byt kratka a popisat vseobecne co funkcia robi. Dalsie vety by to mali vysvetlit podrobnejsie, ale nemusia ist do technickych detialov **ako** je to implementovane, iba **co** funkcia robi. Cize napisat, ze "upravi priponu" namiesto ze "pouzije guess_extension"